acctest/testing.go

```diff
@@ -146,7 +146,7 @@ func Test(t TestT, c TestCase) {
 		},
 		Template: tpl,
 	})
-	err = core.Initialize()
+	err = core.Initialize(packer.InitializeOptions{})
```

Comment:
We can change this yes. We'd change the `Handler` interface to make the arguments a variadic suite of functions, and we can adapt this for both `PackerConfig` and `Core`, I like the idea.

If you don't mind, I think it can be done in a later PR maybe? I don't want to bloat this one with extra changes, there's already a lot going on, but we can open an issue to track this, and we can iterate on the solution after this one is in a satisfactory state?